easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.15.1-foss-2023a.eb

```diff
@@ -0,0 +1,246 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.15.1'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True}
+
+use_pip = True
+sanity_pip_check = True
+
+
+builddependencies = [
+    ('Bazel', '6.1.0'),
+    ('git', '2.41.0', '-nodocs'),
```

Comment:
You seem to have stripped some of the comments from https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.13.0-foss-2023a.eb
I think those are still useful.

Reply:
Thanks for reviewing! I stripped away comments what I was unsure of when tweaking the versions. Will check and put those back.